### ISSUE TITLE
A11y_VSCode_SidebarBarOptions_AccessingTheExtensionbaroptions_ScreenReader: On activating any of the extension under “Popular” and “Recommended” the content in the main content get updated but the same is not notified to the screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode#77064]
**Environment Details:**
VSCode Version : 1.35.6 (user setup)
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 4.1.2
ScreenReader Versions : NVDA2019.1.1, JAWS 2019

**Repro Steps:**
1.	Open Visual studio code editor.
2.	Using "Ctrl+Shift+X" key navigate to the extension side view bar option.
3.	Using "Tab" key navigate to extension side bar options.
4.	Start screen reader.
5.	Navigate to the available extensions under “Popular” and “Recommended” and activate any one of them.
6.	Content in the main content get updated listen if it is announced to the screen reader users.

**Actual:**
When screen reader users navigate and activate the extensions some content in the main content get updated but the same is not notified to the screen reader users.

**Expected:**
The screen reader user should be notified about the updated content on activating any of the extension.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If the screen reader user are not notified about the updated content then the user might not understand what actually happened on activating the extension.

**MAS Reference:**
MAS 1.3.1 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}) 

**Attachment for Reference:**
[VSCode_SidebarBarOptions_AccessingTheExtensionbaroptions_ScreenReader No notification for extension content update.pptx](https://github.com/microsoft/vscode/files/3376377/VSCode_SidebarBarOptions_AccessingTheExtensionbaroptions_ScreenReader.No.notification.for.extension.content.update.pptx)

**Comment by isidorn** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/77062
